### ISSUE TITLE
Should target ratio be read out more when my challenge box is open

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#318]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/582. Noticed with JAWS.
Not so much a bug, but more of a suggestion. When on the second screen, it may be useful for there to be an alert as you move the hands as to what the challenge ratio is and/or a section in the initial scene description that says this. This would only occur if the box was open making the spinners visible. Right now, you need to be reading through the actual box to get the information, which I sometimes found confusing. However, I'm not sure if adding this in is actually a good idea/has been considered already. Feel free to close if this is not desired.

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_all_phet.html
Version: 1.0.0-dev.77 2020-12-08 00:32:10 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by terracoda** [comment]
@KatieWoe, we have indeed discussed this issue.
You are correct that nothing in the state descriptions currently describes the ratio challenge represented in the My Challenge number spinners.

There is a context response when a learner opens My Challenge, and there are responses for both values when either value is changed.

But there is nothing in the state descriptions if My Challenge is open.

Marking as a design issue and assigning to myself.

**Comment by terracoda** [comment]
Here's my suggested design change for the dynamic screen summary:
## Dynamic state-of-sim Screen 2:
**IF My Challenge collapsed:**
Currently, hands, {{at}} challenge ratio and {{very close to each other}}:
- Left hand {{in lower region {{of Play Area}}}}
- Right hand {{in lower-middle region {{of Play Area}}}}

**IF My Challenge is expanded:**
Currently, hands, {{at}} challenge ratio and {{somewhat close to each other}}:
- Left hand {{near bottom {{of Play Area}}}}
- Right hand {{just below middle {{of Play Area}}}}
- Current challenge {{1}} to {{2}} (same phrase as what is used in context responses)

## Static sim-specific interaction hint in a paragraph:
Explore current challenge, or **look for** My Challenge to create your own.

**Comment by terracoda** [comment]
@zepumph, I updated the static help text.

**Comment by terracoda** [comment]
Assigning to you @zepumph to dynamically add the bullet point.

**Comment by zepumph** [comment]
Thanks for taking care of all the static stuff! Please have a look at the new bullet and let me know what you think.

**Comment by terracoda** [comment]
Looks like it working nicely!
Thanks!
